library/IvozProvider/Model/Companies.php

```diff
@@ -154,6 +154,24 @@ public function getMusicClass()
     }
 
     /**
+     * Ensures valid domain value
+     * @param string $data
+     * @return \IvozProvider\Model\Raw\Companies
+     */
+    public function setDomainUsers($data)
+    {
+        if (is_string($data)) {
+            $data = trim($data);
+        }
+
+        if (empty($data)) {
+            throw new \Exception("Domain can't be empty");
```

Comment:
From total ignorance: can we make this text i18n? I've seen it's displayed in the portal.